### PR TITLE
[BRG-456] add checking erc20 token

### PR DESCRIPTION
**Pull request by AlinaLoz**

### CODE REVIEW

File: src/constants/GlobalConstants.js [hunk 1]
```diff
@@ -154,3 +154,14 @@ export const STORE = 'keyval';
...
 export const LENGTH_ID_MESSAGE = 10;
+
+export const ERC20_HASHES = {
+	'totalSupply()': '18160ddd',
+	'balanceOf(address)': '70a08231',
+	'allowance(address,address)': 'dd62ed3e',
+	'transfer(address,uint256)': 'a9059cbb',
+	'approve(address,uint256)': '095ea7b3',
+	'transferFrom(address,address,uint256)': '23b872dd',
+	'Transfer(address,address,uint256)': 'ddf252ad',
```
Comment (stimka) [inline src/constants/GlobalConstants.js]: rm Transfer
add 95d89b41 (symbol) and 313ce567 (decimals)
in my opinion

File: src/helpers/ValidateTransactionHelper.js [hunk 2]
```diff
@@ -73,6 +74,20 @@ class ValidateTransactionHelper {
...
 
+	/**
+	 *
+	 * @param {String} scriptHex
+	 * @returns {boolean}
+	 */
+	static isErc20Contract(scriptHex) {
+		if (scriptHex) {
+			const hashes = Object.values(ERC20_HASHES);
+			return hashes.every((hash) => scriptHex.includes(hash.toString()));
```
Comment (stimka) [inline src/helpers/ValidateTransactionHelper.js]: why toString() needed here?

File: src/constants/GlobalConstants.js (hunk 1 above)
Comment (AlinaLoz) [reply src/constants/GlobalConstants.js]: why? this list option gettin from echo-wallet.

Comment (AlinaLoz) [reply src/constants/GlobalConstants.js]: https://en.wikipedia.org/wiki/ERC-20

**Pull request merged**